chain/actors/builtin/miner/diff.go

```diff
@@ -4,7 +4,8 @@ import (
 	"context"
 
 	cbg "github.com/whyrusleeping/cbor-gen"
-	"go.opentelemetry.io/otel/api/global"
```

Comment:
the bulk of changes in PR are these: upgrading from v0.12 to v0.20 of opentelemetry

Reply:
Wonder if we could do these changes first in a separate PR?

Reply:
Good idea: https://github.com/filecoin-project/lily/pull/749